dask/array/tests/test_random.py

```diff
@@ -274,3 +275,10 @@ def test_create_with_auto_dimensions():
 
         y = da.random.random((10000, 10000), chunks="auto")
         assert y.chunks == ((2500,) * 4, (2500,) * 4)
+
+
+def test_names():
+    name = da.random.normal(0, 1, size=(1000,), chunks=(500,)).name
+
+    assert 'normal' in name
```

Comment:
Should we do `name.startswith("normal-")`?

Reply:
Maybe, if someone changed this to something like `random.normal` or whatever in the future I don't think we should ding them on it.  I really only care that the function name is in the array name and that it is short-ish.  I don't want to ensure that it starts with the word "normal".

Reply:
Couldn't we just change the behavior then as we have done now?

In any event, the test should test the change we have made. FWICT this test passes with or without the change.

Reply:
I agree, the change is actually enforced by the next line

     assert len(key_split(name)) < 10

Reply:
So that would fail if we switched to `random.normal` as well  (in addition to being less clear). Why not just split the key and verify it matches `normal` at the start?

Reply:
It would fail if it was long, yes.  The complaint raised would be that the key they provided was pretty long, which I think is fairly clear.  They can choose to remove that test if they're ok with changing that expectation.

> Why not just split the key and verify it matches normal at the start?

I don't particularly care that it starts with the text "normal".  My concern from before was that it was quite long.  If this is something that you think is valuable though then I'd be happy to add this.

Reply:
We seem to have gone in a circle here.  What would you like done @jakirkham ?  Is it important that we test that the keyname start with `normal`?  I'm more than happy to make this change if it would resolve this concern.

Reply:
Yes, that would resolve it.